package.json

```diff
@@ -67,7 +67,7 @@
     "typescript": "^4.7.4"
   },
   "dependencies": {
-    "antlr4": "~4.8.0",
+    "antlr4": "~4.13.0",
```

Comment:
Why do we want to include an antlr update here? I thought we wanted to continue to avoid updating to the version with performance issues since we did not yet need any other functionality from the updates. Did something change?

Reply:
I did try running the current version of GoFSH and this version against US Core and they both took about the same time, so maybe it doesn't make much of a difference. But I'm just curious if there was anything in particular that convinced you to update antlr4 to 4.13.0 now.

Reply:
I feel like it's simpler to use the same version of `antlr4` in both tools, but you're right that we don't really need any new features.